jquery.balancetext.js

```diff
@@ -282,7 +282,11 @@
     $.fn.balanceText = function (skipResize) {
         var selector = this.selector;
 
-        if (!skipResize && balancedElements.indexOf(selector) === -1) {
+        if (!selector && this[0].className) {
+            selector = "." + this[0].className;
+        }
+
+        if (!skipResize && !!selector && balancedElements.indexOf(selector) === -1) {
```

Comment:
@rileyjshaw jQuery 3.0 no longer supports `.selector`. Do you think this is a reasonable fix?